app/pages/home-for-user/index.jsx

```diff
@@ -120,10 +125,36 @@ const HomePageForUser = React.createClass({
     });
   },
 
+  recentSubjects(ribbonData) {
+    this.setState({
+      newData: this.fetchRecentSubjects(ribbonData),
+    });
+  },
+
   findProjectLink(project) {
     return `/projects/${project.slug}`;
   },
 
+  fetchRecentSubjects(ribbonData) {
+    const newestSubjects = []
+    ribbonData.map((project) => {
+      if (project['recentSubjectSet'] !== undefined){
```

Comment:
Any reason this isn't using dot notation? How about `if (!!project.recentSubjectSet) {`?